src/components/ValidationStatus.vue

```diff
@@ -174,8 +174,10 @@ $button-color: #3BD7FF;
   width: $validation-container-width;
   background-color: $primary-white;
   overflow: auto;
-  margin-bottom: 3rem;
+  margin-bottom: 4.30rem;
   border: 1px solid $border-color;
+  margin-right: 1.20rem;
```

Comment:
Nitpick: If you have trailing 0s, it should be consistent, but it seems we've stuck with not having trailing 0s so I think the above values can be changed to `4.3rem` and `1.2rem`.